### ISSUE TITLE
Columns not referenced in the model

### ISSUE THREAD

**Issue by Laura-GB** [sql-bi/Bravo#212]
### Describe the problem

Columns are highlighted as not used within the model but are being used on visuals. Is that right?
The note of "Before removing them, make sure you are not using these columns in any reports, which Bravo cannot determine" seems to imply other reports rather than check visuals in the current report.

### Bravo version

0.9.8121.26944

### Tabular model

Power BI Desktop, import mode

### Power BI

Power BI Desktop version April 2022

### Steps to reproduce the behavior

1. Look in Analyze

### Screenshots

Start of Week is marked as unused but it is used in a visual
![image](https://user-images.githubusercontent.com/12954710/160442850-95c4323e-13b8-4c4a-9fbf-ace3b482833d.png)

**Comment by darwinlloyd** [comment]
@Laura-GB : I believe unreferenced columns refers that it is not referred in measures not in the visual.

**Comment by marcosqlbi** [comment]
Yes, we have to improve the description of "unreferenced columns". We are evaluating a technique to detect columns referenced in visuals of the same PBIX, but the problem for external reports would remain.
However, we have to improve documentation and description for this version, I get that.

**Comment by RHAWKEOMI** [comment]
@marcosqlbi  
I just stumbled across the same thing, I almost deleted a column in my dataset that is only included in a visual in the same PBIX file and then luckily remember to double check just before. 

I suggest to rename the feature "Unreferenced in Measures" for the time being to not confuse users. 

It would be REALLY nice to be able to get a list of all the columns that are not used in the visuals either.

**Comment by albertospelta** [comment]
I'm closing this issue temporarily and moving to the [Enhancements](https://github.com/sql-bi/Bravo/projects/2) board.

Linking a similar issue https://github.com/sql-bi/Bravo/issues/271

**Comment by tameimpalan** [comment]
2023 and i'm still experiencing this issue.
